Sources/cte/Checker.swift

```diff
@@ -141,6 +141,35 @@ extension Checker {
                 check(node: elsé)
             }
 
+        case .for:
+            let för = node.asFor
```

Comment:
Let's stick to accent acute instead of introducing multiples. It's more keyboard shortcuts to learn otherwise.
`för` -> `fór`